restler/unit_tests/log_baseline_test_files/Bug_Buckets_Json/InvalidValueChecker_500_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": "500",
+    "checkerName": "InvalidValueChecker",
+    "reproducible": true,
+    "verb": "PUT",
+    "endpoint": "/item/{itemName}",
+    "status_text": "Internal Server Error",
+    "request_sequence": [
+        {
+            "requestOrder": 0,
+            "replayrequest": "PUT /item/itemName<test!>-bc8aff2b56?location=centralus&date=2021-01-01 HTTP/1.1\r\nAccept: application/json\r\nHost: unittest\r\nContent-Type: application/json\r\nAUTHORIZATION TOKEN\r\n\r\n{\"datetest\":\"2021-01-01\",\"id\":\"/testparts/{BT\t[\"}\r\n",
```

Comment:
What does request order mean here?

Reply:
request order here mean the order in which the requests in a sequence are executed . ex: there could be a post/put to create and then a delete request is executed and get request is executed 
post/put -0
delete -1
get -2

Reply:
Ok, thank you.  I believe we discussed previously that the request order can be removed since the caller reading this data can use the array index to get the same information.